src/routes/github/create-branch/github-create-branch-options-get.ts

```diff
@@ -26,6 +26,13 @@ export const GithubCreateBranchOptionsGet = async (req: Request, res: Response,
 
 	try {
 		const url = new URL(`${req.protocol}://${req.get("host")}${req.originalUrl}`);
+		if (githubToken && !servers.hasCloudServer && !servers.gheServerInfos.length) {
+			res.render("no-configuration.hbs", {
```

Comment:
You'll need to update the test for this to check we're seeing the correct components based on the conditions.

Reply:
Test case updated